sdk/src/Core/AWSSDK.Core.NetFramework.csproj

```diff
@@ -71,6 +71,7 @@
   <ItemGroup>
     <PackageReference Include="System.Buffers" Version="4.5.1" />
     <PackageReference Include="System.Memory" Version="4.5.5" />
-    <PackageReference Include="System.Text.Json" Version="8.0.5" />
+    <PackageReference Include="System.Text.Json" Version="6.0.11" />
```

Comment:
Can you add a comment here explaining why we are locking to 6.0.11 which will be different then the nuspec file? This seems like something we will forget why in a few years from now.

Reply:
added comment

Reply:
Correction: Windows PowerShell doesn't ship with `System.Text.Json`, its the pwsh. It used to be called PowerShell Core but now its just called PowerShell. You could say `PowerShell(pwsh)`

Reply:
ah okay, thanks for the correction. I will make an update!